src/BuildIntegration/Microsoft.NETCore.Native.Publish.targets

```diff
@@ -66,4 +66,10 @@
     
   </Target>
 
+  <Target Name="CopyNativePdb" Condition="'$(Configuration)'=='Debug'" AfterTargets="Publish">
+    <!-- dotnet CLI produces the incorrect debug symbols - substitute with those we generated during native compilation -->
```

Comment:
Change `incorrect` -> `managed` ? The symbols are produced by dotnet CLI are correct; just not what we want.

Reply:
I debated between the two. Would ```managed-only``` make sense?

Reply:
The common term is "managed PDBs" or "managed symbols". I do not remember anybody saying "managed-only PDBs".

Reply:
Changed.